modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -227,11 +227,10 @@ Simultaneous multithreading (SMT) is enabled by default. If SMT is not enabled i
 ====
 If you disable `hyperthreading`, whether in the BIOS or in the `install-config.yaml`, ensure that your capacity planning accounts for the dramatically decreased machine performance.
 ====
-<4> You must set the value of the `replicas` parameter to `0`. This parameter
-controls the number of workers that the cluster creates and manages for you,
+<4> You must set the value of the `replicas` parameter for the compute machines to `0`. This parameter
+controls the number of compute machines that the cluster creates and manages for you,
 which are functions that the cluster does not perform when you
```

Comment:
Thank you for highlighting this. In installer-provisioned installations, the cluster creates and manages the compute nodes. That function does not apply in user-provisioned deployments, where you install the compute nodes manually. I: have updated that call out as follows, so that this is clearer:

"4. In installer-provisioned installations, this parameter controls the  number of compute machines that the cluster creates and manages for you.  For user-provisioned installations, you must set the value to 0 and manually deploy compute machines for the cluster to use before you finish installing OpenShift Container Platform.

Note: If you are installing a three-node cluster, do not deploy any compute nodes when you create the Red Hat Enterprise Linux CoreOS (RHCOS) machines."